app/services/UpscanService.scala

```diff
@@ -31,7 +30,7 @@ class UpscanService @Inject()(
                              ) {
 
   lazy val redirectUrlBase: String = applicationConfig.upscanRedirectBase
-  private implicit def urlToString(c: Call): String = redirectUrlBase + c.url
+  //private implicit def urlToString(c: Call): String = redirectUrlBase + c.url
```

Comment:
Delete this if is not used. But make sure at runtime the service behave fines in terms of uploading files